staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -92,6 +94,7 @@ func (r *proxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 
 	if !handlingInfo.serviceAvailable {
+		unavailableRequestCounter.WithLabelValues(handlingInfo.name).Inc()
```

Comment:
aren't we already recording status code with sufficient labels to obtain this info? (xref https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go#L157-L170)

if not, should we add API group and version to the existing metric labels, rather than adding a count metric for a specific http code here?

Reply:
The referenced metrics will be recorded in the aggregated apiserver, not here in the aggregator. I.e. we miss those requests. Not sure you mean this: we could add to the very same metrics from the aggregator as well, labelling it with "failed in the aggregator".

Reply:
Added a commit to show what I mean.

Reply:
> What do you think?

That's what I meant above.